### PR TITLE
Selected Text QR Code changes

### PR DESCRIPTION
**Pull request by beatfreaker**
Updated the code such that it will create QR Code for selected text if
any text is selected other wise it will create QR Cod for current Tab's
URL().

### CODE REVIEW

File: popup.js [hunk 1]
```diff
@@ -48,9 +48,29 @@ document.addEventListener("DOMContentLoaded", function () {
...
+            },
+            function(response) {
+                if (response && response.data != "") {
+                    document.getElementById("textbox").value = response.data;
+                    document.getElementById("textbox").select();
+                    updateImage();
+                } else {
+                    document.getElementById("textbox").value = tabs[0].url;
+                    document.getElementById("textbox").select();
+                    updateImage();
```
Comment (fnkr) [inline popup.js]: `document.getElementById("textbox").select()` and `updateImage()` is called in the if and the else block.
Why don't move it to after the if-statement?

File: popup.js [hunk 2]
```diff
@@ -48,9 +48,29 @@ document.addEventListener("DOMContentLoaded", function () {
...
-    document.getElementById("textbox").value = tab.url;
-    document.getElementById("textbox").select();
-    updateImage();
-  });
+  chrome.tabs.query({
+        active: true,
+        currentWindow: true
+    }, function(tabs) {
+        /* ...and send a request for the DOM info... */
+        console.log("test");
```
Comment (fnkr) [inline popup.js]: Please remove any console.* calls!

File: popup.js [hunk 3]
```diff
@@ -48,9 +48,29 @@ document.addEventListener("DOMContentLoaded", function () {
...
-  chrome.tabs.getSelected(null, function(tab) {
-    document.getElementById("textbox").value = tab.url;
-    document.getElementById("textbox").select();
-    updateImage();
-  });
+  chrome.tabs.query({
+        active: true,
+        currentWindow: true
+    }, function(tabs) {
+        /* ...and send a request for the DOM info... */
```
Comment (fnkr) [inline popup.js]: Please remove any unnecessary comments you've copied from stackoverflow.

**Comment by fnkr** [comment]
Hello,

thanks for your contribution!
Please update your pull request, I will merge it then ASAP.

**Comment by beatfreaker** [comment]
Done please check.

File: popup.js (hunk 1 above)
Comment (fnkr) [inline popup.js]: ?

**Comment by fnkr** [comment]
There is still one comment left :)

File: popup.js [hunk 4]
```diff
@@ -48,9 +48,28 @@ document.addEventListener("DOMContentLoaded", function () {
...
-  chrome.tabs.getSelected(null, function(tab) {
-    document.getElementById("textbox").value = tab.url;
-    document.getElementById("textbox").select();
-    updateImage();
-  });
+  chrome.tabs.query({
+        active: true,
+        currentWindow: true
+    }, function(tabs) {
+        console.log("test");
```
Comment (fnkr) [inline popup.js]: Please remove this too!

File: popup.js [hunk 5]
```diff
@@ -48,9 +48,28 @@ document.addEventListener("DOMContentLoaded", function () {
...
+        console.log(tabs);
```
Comment (fnkr) [inline popup.js]: Please remove this too!

File: selection.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
+chrome.runtime.onMessage.addListener(function(msg, sender, response) {
+console.log("called here");
```
Comment (fnkr) [inline selection.js]: Please remove this too!

File: popup.js (hunk 4 above)
Comment (fnkr) [inline popup.js]: Please remove any console.* calls!

**Comment by fnkr** [comment]
There are still comments left. [Check the files view](https://github.com/fnkr/chrome-qrcode-extension/pull/2/files).

**Comment by beatfreaker** [comment]
Done .... only following comment left
// ignore and try to use bigger QR code format
and i guess it was added by you .... i should remove that too??

**Comment by fnkr** [comment]
[Yes, its mine](https://github.com/fnkr/chrome-qrcode-extension/blob/54f2dc8/popup.js#L16). Leave it.

I will test your PR ASAP and merge it if everything works.

**Comment by fnkr** [comment]
Merged it, thanks for your contribution!
Chrome webstore may needs up to 60 minutes until its live.

**Pull request merged**